server.js

```diff
@@ -1,9 +1,12 @@
 const express = require('express')
+const route = require('./route')
 const app = express()
 const mongoose = require('mongoose')
 const URI = 'mongodb+srv://dhruvk:q6GkQB4Kg3jvSHFd@dhruv-bike-info.dga5pzc.mongodb.net/?retryWrites=true&w=majority'
```

Comment:
The MongoDB URI should not be hard-coded and exposed in the code. It should be stored in an environment variable for security reasons. [important]